### PR TITLE
Update Order.java

### PR DESCRIPTION
**Pull request by Priyankagowda0707**
used lombok annotations . pls review changes

### CODE REVIEW

File: java/src/Order.java [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 import java.util.List;
 
+@Getter
+@Setter
+//@Data
```
Comment (abhilashgowda23) [inline java/src/Order.java:7]: remove Data annotation. @Priyankagowda0707

**Comment by Priyankagowda0707** [comment]
done.

**Review by abhilashgowda23 — APPROVED** [review]
done.

**Comment by abhilashgowda23** [comment]
approved.

**Pull request merged**